src/checks.rs

```diff
@@ -1650,6 +1655,9 @@ impl CheckKind {
                 "Unnecessary parameters to `functools.lru_cache`".to_string()
             }
             CheckKind::UnnecessaryEncodeUTF8 => "Unnecessary call to `encode` as UTF-8".to_string(),
+            CheckKind::UnnecessaryTypedDictSyntacticForm => {
+                "Unnecessary use of syntactic form of TypedDict".to_string()
```

Comment:
“Unnecessary” implies that something could be deleted; both forms are “syntactic”.

I suggest something like `` Convert `TypedDict` functional syntax to class syntax ``.

Also, this should all apply equally to `NamedTuple`.

Reply:
Yeah agreed, nice suggestion.